nncf/pruning/filter_pruning/algo.py

```diff
@@ -29,6 +29,7 @@
 from nncf.pruning.utils import get_rounded_pruned_element_number
 
 
+# pylint:disable=useless-super-delegation
 @COMPRESSION_ALGORITHMS.register('filter_pruning')
 class FilterPruningBuilder(BasePruningAlgoBuilder):
     def __init__(self, config, should_init: bool = True):
```

Comment:
should be resolved by deleting `__init__` method

Reply:
Done